go.mod

```diff
@@ -38,7 +38,7 @@ require (
 	github.com/hashicorp/serf v0.9.0 // indirect
 	github.com/influxdata/influxdb v1.8.0
 	github.com/jpillora/backoff v1.0.0 // indirect
-	github.com/json-iterator/go v1.1.9
+	github.com/json-iterator/go v1.1.10
```

Comment:
ideally we upgrade those in separate PR. Or at least you can mention in commit `update client_golang` (:

Reply:
Changelog added.

Reply:
Hmm, I don't think we should mention it in the CHANGELOG. I think @bwplotka was referring to the commit description. Furthermore, this is about json-iterator, not client_golang. But I guess it was _triggered_ by the client_golang update (which is normal with the way Go modules work).

So I'd say just add "Updated prometheus/client_golang and json-iterator/go" to the commit description (not the changelog).

Reply:
Yes, I meant commit message (:

Reply:
Sorry I misunderstood :(

PR updated, PTAL.